Source/ChocolateyGui.Common.Windows/Bootstrapper.cs

```diff
@@ -114,7 +114,7 @@ protected override async void OnStartup(object sender, StartupEventArgs e)
             {
                 // Do not remove! Load Chocolatey once so all config gets set
                 // properly for future calls
-                var choco = Lets.GetChocolatey();
+                var choco = Lets.GetChocolatey(false);

```

Comment:
For the bare boolean parameter here, can you use the named parameter syntax?

```suggestion
                var choco = Lets.GetChocolatey(initializeLogging: false);
```